client/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <translations>
-   <translation id="telephony_line_phone_order_title" qtlid="336844">Commander un Plug &amp; Phone</translation>
-   <translation id="telephony_line_phone_order_change_title" qtlid="458858">Changer de téléphone Plug &amp; Phone</translation>
+   <translation id="telephony_line_phone_order_title" qtlid="336844">Commander un Plug & Phone</translation>
```

Comment:
Could you please stay with `&amp;`? That's a requirements when using translations scripts.